dias/chime_analyzer.py

```diff
@@ -16,6 +16,11 @@ class CHIMEAnalyzer(Analyzer):
     archive_data_dir = config.Property(proptype=str)
     staging_data_dir = config.Property(proptype=str)
 
+    if not os.path.exists(archive_data_dir):
+        raise OSError("archive directory ({}) is missing".format(archive_data_dir))
+    if not os.path.exists(staging_data_dir):
+        raise OSError("staging directory ({}) is missing".format(staging_data_dir))
```

Comment:
**marimba** has Python 3.7.9, so we should be able to use f-strings in the future! No need to make that change now.

Reply:
I'm happy so long as we aren't using `%` anymore.